File: sdk/dart/lib/model/product_assignment.dart
part of swagger.api;


@Entity()
class ProductAssignment {
  
  @Property(name: 'ProductID')
  String productID = null;
  

  @Property(name: 'StandardPriceScheduleID')
  String standardPriceScheduleID = null;
  

  @Property(name: 'ReplenishmentPriceScheduleID')
  String replenishmentPriceScheduleID = null;
  

  @Property(name: 'UserID')
  String userID = null;
  

  @Property(name: 'UserGroupID')
  String userGroupID = null;
  

  @Property(name: 'BuyerID')
  String buyerID = null;
  
  ProductAssignment();

  @override
  String toString()  {
    return 'ProductAssignment[productID=$productID, standardPriceScheduleID=$standardPriceScheduleID, replenishmentPriceScheduleID=$replenishmentPriceScheduleID, userID=$userID, userGroupID=$userGroupID, buyerID=$buyerID, ]';
  }

}

File: sdk/async-scala/src/main/scala/io/swagger/client/model/ProductAssignment.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class ProductAssignment (
  ProductID: Option[String],
StandardPriceScheduleID: Option[String],
ReplenishmentPriceScheduleID: Option[String],
UserID: Option[String],
UserGroupID: Option[String],
BuyerID: Option[String]
)
